File: Initial Prototype/Client/MyApplicationProject/MyApplication/src/main/java/com/example/myapplication/dtos/JourneyMessageMarkerDTO.java
package com.example.myapplication.dtos;

/**
 * Created by Michal on 13/02/14.
 */
public class JourneyMessageMarkerDTO {
    public int UserId;
    public int JourneyMessageId;

    public JourneyMessageMarkerDTO(int userId, int journeyMesssageId) {
        this.UserId = userId;
        this.JourneyMessageId = journeyMesssageId;
    }
}

File: Project/WebService/Services/DTOs/JourneyMessageMarkerDTO.cs
// --------------------------------------------------------------------------------------------------------------------
// <copyright file="JourneyMessageMarkerDTO.cs" company="">
//   
// </copyright>
// <summary>
//   The journey message marker dto.
// </summary>
// --------------------------------------------------------------------------------------------------------------------

namespace Services.DTOs
{
    using System.Runtime.Serialization;

    /// <summary>
    /// The journey message marker dto.
    /// </summary>
    [DataContract]
    public class JourneyMessageMarkerDTO
    {
        /// <summary>
        /// Gets or sets the journey message id.
        /// </summary>
        [DataMember]
        public int JourneyMessageId { get; set; }

        /// <summary>
        /// Gets or sets the user id.
        /// </summary>
        [DataMember]
        public int UserId { get; set; }
    }
}